src/app/constants/time.ts

```diff
@@ -0,0 +1 @@
+export const dayInSeconds = 24 * 60 * 60
```

Comment:
/nit: DAY_IN_SECONDS (upper case) for constants

Reply:
07fa436af